docs/site/Parsing-requests.md

```diff
@@ -208,9 +208,33 @@ A full list of validation keywords could be found in the
 
 ##### Custom Error Messages
 
-You can also specify custom error messages for the JSON schema validation rules
-in the model property decorator. The messages are added in field called
-`errorMessage` inside `jsonSchema` like:
+You can also specify the validation rules in the `jsonSchema` field of the
```

Comment:
+1 to update all relevant places.

It makes me wonder though, why are we duplicating this content? Can we change one of these two pages and replace "custom error messages" section with a link to the other page?

For example:

```diff
- ##### Custom Error Messages
- You can also specify custom error messages for the JSON schema validation rules
- (...)
+ You can also specify custom error messages for JSON schema validation rules, as explained in
+ [Custom Validation Rules and Error Messages](Model.md#custom-validation-rules-and-error-messages).
```